### PR TITLE
NO-ISSUE: Remove almost all references to `kiegroup`

### PR DESCRIPTION
**Pull request by tiagobento**
Some references I couldn't remove yet, here are some of them:
- dmn-feel-handbook
- `kiegroup/kie-samples` on SLWT
- `quay.io/kiegroup` for base-builder (`kogito-swf-{devmode,builder}` will be copied here, see https://github.com/apache/incubator-kie-issues/issues/1001)
- `kiegroup/kogito-serverless-operator` (will also be copied here, see https://github.com/apache/incubator-kie-issues/issues/1034)
- `kiegroup/serverless-logic-sandbox-deployment` @ `quarkus-accelerator` on SLWT
- sonar.organization on some Maven packages
- Reference to when the BPMN Marshaller was last synced manually with the backend.

Some important changes include:
1. The new default namespace for DMN files (`https://kie.apache.org/dmn/[UUID]`)
2. The namespace of the DMN extensions changed to `https://kie.apache.org/dmn/extensions/1.0` (we have a migration that takes care of updating it when upgrading DMN 1.2 files to DMN 1.3 and above.

### CODE REVIEW

**Review by thiagoelg — APPROVED** [review]

File: packages/dashbuilder/appformer/pom.xml [hunk 1]
```diff
@@ -82,11 +82,12 @@
     <netty-transport-native-epoll-classifier>linux-x86_64</netty-transport-native-epoll-classifier>
     <netty-transport-native-kqueue-classifier>osx-x86_64</netty-transport-native-kqueue-classifier>
 
-    <version.bundle.plugin
```
Comment (porcelli) [inline packages/dashbuilder/appformer/pom.xml:85]: Probably this can be reverted

File: packages/dashbuilder/appformer/pom.xml [hunk 2]
```diff
@@ -82,11 +82,12 @@
...
-    >3.3.0</version.bundle.plugin><!-- 3.2.0 sometimes 
-      fails with NPE during parallel build -->
+    <!-- 3.2.0 sometimes fails with NPE during parallel build -->
+    <version.bundle.plugin>3.3.0</version.bundle.plugin>
+
     <!-- Make OSGi happy -->
     <osgi.snapshot.qualifier>${maven.build.timestamp}</osgi.snapshot.qualifier>
+
     <jboss.releases.repo.url
     >https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</jboss.releases.repo.url>
```
Comment (porcelli) [inline packages/dashbuilder/appformer/pom.xml:92]: isn't Apache staging better repo to use?

File: packages/dashbuilder/appformer/pom.xml [hunk 3]
```diff
@@ -345,7 +346,7 @@
           <version>1.3</version>
         </plugin>
 
-        <!--This plugin's configuration is used to store Eclipse m2e settings 
+        <!-- This plugin's configuration is used to store Eclipse m2e settings
```
Comment (porcelli) [inline packages/dashbuilder/appformer/pom.xml:349]: unnecessary change :)

File: packages/dashbuilder/appformer/pom.xml [hunk 4]
```diff
@@ -439,7 +440,7 @@
         </configuration>
       </plugin>
 
-      <!--Fix translations from Zanata -->
+      <!-- Fix translations from Zanata -->
```
Comment (porcelli) [inline packages/dashbuilder/appformer/pom.xml:443]: Unnecessary change?

File: packages/dmn-editor/stories/misc/empty/Empty.stories.tsx [hunk 5]
```diff
@@ -29,7 +29,7 @@ export const generateEmptyDmn15 = () => `<?xml version="1.0" encoding="UTF-8"?>
 <definitions
   xmlns="${dmn15ns.get("")}"
   expressionLanguage="${DMN15_SPEC.expressionLanguage.default}"
-  namespace="https://kie.org/dmn/${generateUuid()}"
+  namespace="https://kie.apache.org/dmn/${generateUuid()}"
```
Comment (porcelli) [inline packages/dmn-editor/stories/misc/empty/Empty.stories.tsx:32]: will this break users coming from precisou version and using new editor?

how this affects runtime engine?

File: packages/dmn-marshaller/src/schemas/dmn-1_5/Dmn15Spec.ts [hunk 6]
```diff
@@ -101,4 +101,4 @@ export const allDmnImportNamespaces = new Set([
   dmn15ns.get("")!,
 ]);
 
-export const KIE_DMN_UNKNOWN_NAMESPACE = "https://kie.org/dmn/unknown";
+export const KIE_DMN_UNKNOWN_NAMESPACE = "https://kie.apache.org/dmn/unknown";
```
Comment (porcelli) [inline packages/dmn-marshaller/src/schemas/dmn-1_5/Dmn15Spec.ts:104]: same question: how this affects users from previous version using new editor

File: packages/kn-plugin-workflow/pkg/command/quarkus/build.go [hunk 7]
```diff
@@ -21,13 +21,14 @@ package quarkus
 
 import (
 	"fmt"
+	"regexp"
```
Comment (porcelli) [inline packages/kn-plugin-workflow/pkg/command/quarkus/build.go:24]: Is this change needed?

**Review by porcelli — COMMENTED** [review]
thank you for the very diligent work!

File: packages/dmn-marshaller/src/schemas/dmn-1_5/Dmn15Spec.ts (hunk 6 above)
Comment (tiagobento) [reply packages/dmn-marshaller/src/schemas/dmn-1_5/Dmn15Spec.ts:104]: Users coming from the legacy DMN Editor will be on DMN 1.2, and we migrate all models to 1.5 as soon as a change is made to them using the new DMN Editor. One of those migrations is exactly changing this namespace.

For the engine, this is ignored, as the extensions we have have no semantics for running. See https://github.com/apache/incubator-kie-tools/blob/main/packages/dmn-marshaller/src/schemas/kie-1_0/KIE.xsd

File: packages/dmn-editor/stories/misc/empty/Empty.stories.tsx (hunk 5 above)
Comment (tiagobento) [reply packages/dmn-editor/stories/misc/empty/Empty.stories.tsx:32]: No.. this is an arbitrary string that is user-defined.

File: packages/kn-plugin-workflow/pkg/command/quarkus/build.go (hunk 7 above)
Comment (tiagobento) [reply packages/kn-plugin-workflow/pkg/command/quarkus/build.go:24]: Auto-formatting..

File: packages/dashbuilder/appformer/pom.xml (hunk 4 above)
Comment (tiagobento) [reply packages/dashbuilder/appformer/pom.xml:443]: Auto-formatting...

File: packages/dashbuilder/appformer/pom.xml (hunk 3 above)
Comment (tiagobento) [reply packages/dashbuilder/appformer/pom.xml:349]: Auto-formatting...

File: packages/dashbuilder/appformer/pom.xml (hunk 2 above)
Comment (tiagobento) [reply packages/dashbuilder/appformer/pom.xml:92]: No idea! If it is, we can send another PR.

File: packages/dashbuilder/appformer/pom.xml (hunk 1 above)
Comment (tiagobento) [reply packages/dashbuilder/appformer/pom.xml:85]: Oh, this file gave me a lot of trouble with auto-formatting, but I left it life this because the formatter is happier :)

File: packages/dashbuilder/appformer/pom.xml (hunk 2 above)
Comment (porcelli) [reply packages/dashbuilder/appformer/pom.xml:92]: mind share an issue to be followed up? :)

File: packages/kn-plugin-workflow/pkg/command/quarkus/build.go (hunk 7 above)
Comment (porcelli) [reply packages/kn-plugin-workflow/pkg/command/quarkus/build.go:24]: 👍

**Review by porcelli — APPROVED** [review]
thank you again for the diligent work!

File: packages/dashbuilder/appformer/pom.xml (hunk 2 above)
Comment (tiagobento) [reply packages/dashbuilder/appformer/pom.xml:92]: https://github.com/apache/incubator-kie-issues/issues/1071
